CMakeLists.txt

```diff
@@ -291,6 +305,16 @@ FILES_MATCHING
 # copy cmake files
 set(INSTALL_CMAKE_DIR lib/CMake/ETISS CACHE PATH "Installation directory for CMake files")
 
+# to allow linking with ETISS lib in build-tree
+# copying headers
+#file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/include/ DESTINATION ${ETISS_BINARY_DIR}/include/)
+#file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/include_c/ DESTINATION ${ETISS_BINARY_DIR}/include/)
+file(GLOB JIT_HEADERS ${CMAKE_CURRENT_SOURCE_DIR}/src/jitlibs/**/*.h)
+file(COPY ${JIT_HEADERS} DESTINATION ${ETISS_BINARY_DIR}/include/jit/etiss/jit/)
+# plugins/list.txt file
+file(WRITE ${ETISS_BINARY_DIR}/lib/plugins/list.txt)
```

Comment:
this way it will be empty right? what is the consequence?

Reply:
the file is only created here, it is populated during calls to `ETISSPlugin`, defined in `cmake/ETISSPlugin`.

Reply:
there are actually a few places that still used `ETISS_BINARY_DIR` even though it was removed at some point.

makes me wonder if the code on master actually does what is expected from it.

besides ETISSPlugin.cmake it was also used here: https://github.com/tum-ei-eda/etiss/blob/32ddb52052df07f4241dfc7fda64fa671b72acd1/cmake/RegisterJITFiles.cmake#L14